packages/auth-otp/plugin.ts

```diff
@@ -31,15 +35,30 @@ interface OtpAuthConfig extends Zemble.GlobalConfig {
    * email, name and two factor code.
     */
   readonly emailHtml?: string
-  readonly handleAuthRequest?: (email: IEmail, twoFactorCode: string, context: Zemble.GlobalContext) => Promise<void> | void
-  readonly generateTokenContents: ({ email }: {readonly email: string}) => Promise<Omit<Zemble.OtpToken, 'iat'>> | Omit<Zemble.OtpToken, 'iat'>
+
+  // SMS
+
+  readonly fromSms?: string
+  /*
+   * Write {{twoFactorCode}} to have them replaced with the
+   * email, name and two factor code.
+    */
+  readonly smsMessage?: string
+  readonly WHITELISTED_COUNTRY_CODES?: readonly CountryCode[]
+
+  readonly handleEmailAuthRequest?: (email: IEmail, twoFactorCode: string, context: Zemble.GlobalContext) => Promise<void> | void
+  readonly handleSmsAuthRequest?: (phoneNum: string, twoFactorCode: string, context: Zemble.GlobalContext) => Promise<void> | void
+  readonly generateTokenContents: ({ emailOrPhone }: {readonly emailOrPhone: string}) => Promise<Omit<Zemble.OtpToken, 'iat'>> | Omit<Zemble.OtpToken, 'iat'>
```

Comment:
It's probably better to send it in as `{ email?: string, phoneNumber?: PhoneNumber }` so the developer consuming the API doesn't have to check if it's an email or phone number again